src/concrete/ml/quantization/quantized_module.py

```diff
@@ -632,6 +632,7 @@ def compile(
         global_p_error: Optional[float] = None,
         verbose: bool = False,
         inputs_encryption_status: Optional[Sequence[str]] = None,
+        compress_evaluation_keys: bool = True,
```

Comment:
I would rather we set this through the configuration and keep only ML related params here

Reply:
then, would be awesome to have a single place / class for crypto-related configs, and we would change just one place and it would apply to all models we support (linear, tree, qnn, torch). Maybe already the case!